internal/util/vault_tokens.go

```diff
@@ -50,6 +51,73 @@ const (
 	vaultTokenSecretKey = "token"
 )
 
+type standardVault struct {
+	KmsPROVIDER        string `json:"KMS_PROVIDER"`
+	VaultADDR          string `json:"VAULT_ADDR"`
```

Comment:
the contents and valid keys for the ConfigMap need to be documented somewhere

Reply:
i will add a new document for this configuration in a followup PR

Reply:
ok! please add a note in the [KMS VaultTokens project](https://www.lenovo.com/nl/nl/nlcepp) or create an issue for it.

Reply:
done https://github.com/ceph/ceph-csi/issues/1793